apps/web/src/app/[locale]/(dashboard)/[wsId]/(workspace-settings)/migrations/migration-dashboard.tsx

```
@@ -136,325 +177,236 @@ export default function MigrationDashboard() {
+      ) {
+        await fetchData(
+          `${externalUrl}?from=${externalData.length}&limit=${chunkSize}`.replace(
+            /\?([^?]*)(\?)/g,
+            '?$1&'
+          ),
+          {
+            onSuccess: (newData) => {
+              if (externalCount === -1) externalCount = newData.count;
+              externalData = [
+                ...externalData,
+                ...newData?.[externalAlias ?? internalAlias ?? 'data'],
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)